packages/teams-js/src/public/chat.ts

```diff
@@ -67,20 +73,22 @@ export namespace chat {
    * @beta
    */
   export function openChat(openChatRequest: OpenSingleChatRequest): Promise<void> {
+    const apiVersionTag = getApiVersionTag(chatTelemetryVersionNumber, ApiName.Chat_OpenChat);
     return new Promise<void>((resolve) => {
       ensureInitialized(runtime, FrameContexts.content, FrameContexts.task);
       if (!isSupported()) {
         throw errorNotSupportedOnPlatform;
       }
       if (runtime.isLegacyTeams) {
         resolve(
-          sendAndHandleStatusAndReason(
+          sendAndHandleStatusAndReasonWithVersion(
```

Comment:
>sendAndHandleStatusAndReasonWithVersion [](http://example.com/codeflow?start=10&length=39)

When I was looking at PR #2053, I assumed (incorrectly) that you weren't tagging the `executeDeepLink` versions of these, only the function-based versions of these. I had made this assumption in that PR because I saw that in `calendar.ts`, you didn't apply a telemetry tag to the `composeMeeting` branch that called `executeDeepLink`, only the function one.

In retrospect, it was silly of me to make that assumption because in that same PR you _did_ add a tag to the `executeDeepLink` version of the `startCall` function (as well as elsewhere).

That was a long-winded way of saying -- should the `executeDeepLink` version of `composeMeeting` from the previous PR have gotten tagged?

Reply:
Sounds great.